src/Evolution/Systems/ScalarWave/BoundaryConditions/ConstraintPreservingSphericalRadiation.hpp

```diff
@@ -153,6 +154,11 @@ convert_constraint_preserving_spherical_radiation_type_from_yaml(
  * \f$-\partial_i\Pi\f$, which is the evolution equation but without the
  * constraint.
  *
+ * \note On a moving mesh the characteristic speeds change according to
+ * \f$\lambda\to\lambda-v^i_gn_i\f$ where \f$v^i_g\f$ is the mesh velocity.
```

Comment:
I don't quite understand what you mean with this `\to` notation here. Do you mean it in the sense of "on a moving mesh the char speeds have this extra term"? If so, why is (for example) the `dt_psi_correction` zero for a non-moving mesh (that's what the current implementation does I think), i.e. what happened to the term `\lambda_\Psi n^i C_i`?

Reply:
Yes, that's right. I've seen this notation a lot, basically "Do the substitution `x\to x+y`".
The psi correction is added on lines 127 and 132, unless I'm not understanding what you mean.

Reply:
Yes the correction from the moving mesh is added on lines 127 and 132. For a non-moving mesh the correction is zero though (line 156). That makes me wonder what happened to the non-moving-mesh term in `(\lambda-v^i_gn_i) n^i C_i`.

Reply:
`\lambda_psi` is identically zero

Reply:
right, flat scalar wave 👍